index.js

```diff
@@ -521,6 +521,54 @@
       [$.Function, b, a, c],
       function(f, x, y) { return f(y, x); });
 
+  //# map :: Functor f => (a -> b) -> f a -> f b
+  //.
+  //. Takes a function and a [Functor][], applies the function to each of the
+  //. Functor's values, and returns a Functor of the same shape.
+  //.
+  //. ```javascript
+  //. > S.map(S.inc, [1, 2, 3])
+  //. [2, 3, 4]
+  //.
+  //. > S.map(S.inc, {a: 1, b: 2, c: 3})
+  //. {a: 2, b: 3, c: 4}
+  //.
+  //. > S.map(S.inc, S.Just(2))
+  //. S.Just(3)
+  //.
+  //. > S.map(S.inc, S.Nothing())
+  //. S.Nothing()
+  //.
+  //. > S.map(S.not, S.odd)(2)
+  //. true
```

Comment:
Yeah, it's pretty neat. I wonder should I remove the `S.nothing` example there, there's probably no need for two `Maybe` examples.